lib/util.js

```diff
@@ -1069,7 +1076,8 @@ module.exports = {
 
           param = new sdk.QueryParam({
             key: key,
-            value: value
+            value: value,
+            description: description
```

Comment:
@Dhroov7 check if we can do param.description=description after the constructor - runtime will take time to fix, and I want to get this release out.

Reply:
if that works, add a comment explaining why it was done - i think there's already one other place where we do this

Reply:
@Dhroov7 need updates here